sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline.py

```diff
@@ -2891,3 +2891,19 @@ def register_both_output():
         check_name_and_version(pipeline_job.outputs.pipeine_b_output, "n2_o_output", "2")
         check_name_and_version(pipeline_job.jobs["node_3"].outputs["component_out_path"], "n3_o_output", "4")
         check_name_and_version(pipeline_job.jobs["sub_node"].outputs["sub_pipeine_a_output"], "subgraph_o_output", "1")
+
+    def test_pipeline_input_binding_limits_timeout(self, client: MLClient):
+        component_yaml = r"./tests/test_configs/components/helloworld_component_no_paths.yml"
```

Comment:
What's the behavior before? I thought this should be supported in `support_data_binding_expression_for_fields` in `BaseNodeSchema`'s init

Reply:
Currently it's still a PipelineInput and will raise error since there is iso date transition logic. Have passed that as we discussed offline

Reply:
Looks the issue comes from trying to do iso time transformation for data-binding expression in `_from/to_rest_object`. With `support_data_binding_expression_for_fields`, a PipelineInput will be dumped to a data-binding expression in string, but will still fail when trying to transfer `${{parent.inputs.xxx}}` to an iso time format string.

@0mza987 please correct me if it's not the case

Reply:
Yes, that's what happened here.